CONTRIBUTING.md

```diff
@@ -28,17 +28,19 @@ Let's take a look at what it takes to contribute with **A)**:
 
     Before a Pull Request can be created, it is required to have an issue registered, which has been approved for development by Microsoft. This is done to ensure that community members aren't wasting their time on writing code, which later cannot be accepted.
 
-2. **Wait for approval** 
+2. **Wait for approval**
 
     Now that your issue is registered, Microsoft will triage your issue and either approve or reject it. An approved issue will have a label "approved".
 
-3. **Create a Pull Request** 
+3. **Create a Pull Request**
+
+    Now a Pull Request can be created. It is important that the Pull Request is linked to an approved issue. **Pull Requests which aren't linked to approved issues will get rejected.**
 
-    Now a Pull Request can be created. It is important that the Pull Request is linked to an approved issue. **Pull Requests which aren't linked to approved issues will get rejected.** 
-    
    If it is your first contribution, you are required to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com. When you submit a pull request, a CLA-bot will automatically determine whether you need
     to provide a CLA and decorate the Pull Request appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repositories using our CLA.
 
+    Before creating a Pull Request, you might want to verify your changes work as expected. Refer to [Local Development Environment guidelines](LOCAL_DEV_ENV.md) for more info.
```

Comment:
Can you please add some info for draft PRs? Can we use them? Any special rules? (in the baseapp repo the name must contain [DRAFT] etc.)?

Reply:
There are no special rules at this point. The same validations work on draft PRs as well, so no real differentiation.